packages/create-turbo/src/commands/create/index.ts

```diff
@@ -0,0 +1,231 @@
+import path from "path";
+import chalk from "chalk";
+import type { Project } from "@turbo/workspaces";
+import {
+  getWorkspaceDetails,
+  install,
+  getPackageManagerMeta,
+  ConvertError,
+} from "@turbo/workspaces";
+import { getAvailablePackageManagers } from "turbo-utils";
+import type { CreateCommandArgument, CreateCommandOptions } from "./types";
+import * as prompts from "./prompts";
+import { createProject } from "./createProject";
+import { tryGitCommit, tryGitInit } from "../../utils/git";
+import { isOnline } from "../../utils/isOnline";
+import { transforms } from "../../transforms";
+import { turboGradient, turboLoader, info, error, warn } from "../../logger";
+
+function handleWorkspaceErrors(err: unknown) {
+  if (err instanceof ConvertError && err.type !== "unknown") {
+    error(chalk.red(err.message));
+    process.exit(1);
+  } else {
+    // if it's an unknown error type, re-throw to root
+    throw err;
+  }
+}
+
+const SCRIPTS_TO_DISPLAY: Record<string, string> = {
+  build: "Build",
+  dev: "Develop",
+  test: "Test",
+  lint: "Lint",
+};
+
+export async function create(
+  directory: CreateCommandArgument,
+  packageManager: CreateCommandArgument,
+  opts: CreateCommandOptions
+) {
+  const { skipInstall, skipTransforms } = opts;
+  console.log(chalk.bold(turboGradient(`\n>>> TURBOREPO\n`)));
+  info(`Welcome to Turborepo! Let's get you set up with a new codebase.`);
+  console.log();
+
+  const availablePackageManagers = await getAvailablePackageManagers();
+  const online = await isOnline();
+  if (!online) {
+    error(
+      "You appear to be offline. Please check your network connection and try again."
+    );
+    process.exit(1);
+  }
+  const { root, projectName } = await prompts.directory({ directory });
+  const relativeProjectDir = path.relative(process.cwd(), root);
+  const projectDirIsCurrentDir = relativeProjectDir === "";
+
+  // selected package manager can be undefined if the user chooses to skip transforms
+  const selectedPackageManagerDetails = await prompts.packageManager({
+    packageManager,
+    skipTransforms,
+  });
+
+  if (packageManager && opts.skipTransforms) {
+    warn(
+      '"--skip-transforms" flag conflicts with the package manager argument. The package manager argument will be ignored.'
+    );
```

Comment:
You can use the `.conflicts()` method in commander instead of this https://github.com/tj/commander.js/blob/HEAD/examples/options-conflicts.js

Reply:
I did look at that originally, but package manager is an argument, and skip-transform is an option, and you can't specify conflicts between the two.